utils/resourceBuilder.ts

```diff
@@ -42,6 +42,9 @@ export class ResourceBuilder {
     if (this.tags == null || this.tags === undefined || this.tags.length === 0) {
       throw new ResourceBuilderError('Unable to build incomplete Resource')
     }
+    if (this.blockchain == null || this.blockchain === undefined || this.blockchain.length === 0) {
+      throw new ResourceBuilderError('Unable to build incomplete Resource')
+    }
```

Comment:
Looking at the [Add Resource form on Airtable](https://airtable.com/shrJqjAqVpGgDa47t) Blockchain is not a required field, so we should be able to create a Resource without an entry for it.

Reply:
good catch.

Reply:
I liked the idea that @with-heart had in having the arrays initialized upon creation, but are empty.

so I will just take the `this.blockchian.length ===0` out, and leave the null/undefined if you have no objections.

Reply:
scratch that, I thought this was building a contributor. I don't think it's necessary for blockchain to be a required resource. Whoops!